src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -4431,6 +4479,139 @@ public record Derived(bool unused) : Base(unused);
         );
     }
 
+    [Fact, CompilerTrait(CompilerFeature.NullableReferenceTypes)]
+    public void RequiredMemberSuppressesNullabilityWarnings_ChainedConstructor_11()
```

Comment:
This test is unchanged: just moving it back to be in line with the rest of the tests with the same name.